featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -130,7 +130,8 @@ def progress_callback(*args):
             if missing_ids:
                 default_df = self.generate_default_df(instance_ids=missing_ids,
```

Comment:
Would it make sense to warn the user that there are invalid instance IDs?

Reply:
Since this isn't directly related to the initial but, will create a new issue to track.